### ISSUE TITLE
[Suggestion] Add an option for fixed effect timing

### ISSUE THREAD

**Issue by TyroneSama** [Drewol/unnamed-sdvx-clone#273]
I play at global offset 100—charts that use separate effected .oggs play pretty much perfectly, but charts that use dynamic effects have off-sync effect audio despite input timing feeling perfect. Slam sounds, hitsounded FX notes, and filters all start/end noticeably delayed from the music, which can be really distracting in rapid slam sections in particular.

Would it be possible to add an option to play dynamic effects at the "correct" timing regardless of player input? This would probably include playing hitsounds/effects even on complete misses, but that seems preferable to off-sync effects, and I don't know if there's a better option for players on high offset.

(I apologize if this is completely unintelligible, I don't really hang out in the SDVX sim community and probably don't know the correct names for most charting-related things.)

**Comment by Drewol** [comment]
Yeah, it wouldn't be an option as the global offset affecting effects is just a straight up bug that has been present for a long time. I made an attempt to fix it some time ago but with no success so for now I just recommend trying to keep your global offset as close to 0 as possible.

**Comment by TyroneSama** [comment]
Shit. Well, guess it's time for me to start screwing with my audio or learning to play off-sync. Good luck, thank you for the prompt response!

**Comment by Drewol** [comment]
100ms seems on the high end, global offset is only really intended to fix audio-visual sync and input offset should be used when things appear in sync.
